### PR TITLE
Update whats-new.md

### PR DESCRIPTION
**Pull request by caw310**
Updated with Fall 2024 updates

### CODE REVIEW

File: src/_about/whats-new.md [hunk 1]
```diff
@@ -25,6 +20,45 @@ title: What’s new?
...
+* [Design] Table variations #3137 
+* [Design]Update Alert spacing in Figma #3231 
+* Update breadcrumb spacing in Page templates - Design #3049 
+* Update guidance for Keep a record of submitted information pattern #3054 
+* Add List guidance to VADS #3206 
+* Alerts - Sign in - Documentation 
+* Issue with downloading logos (PNG format) 
+* [Documentation] - Update va-link External link variation guidance #3224
+* [Documentation] - Multiple responses pattern and flow chart #3219
+* [Documentation] - Update guidance for select component - Documentation #3251
```
Comment (babsdenney) [inline src/_about/whats-new.md:58]: Documentation is said twice

File: src/_about/whats-new.md [hunk 2]
```diff
@@ -25,6 +20,45 @@ title: What’s new?
...
+* [Documentation]Guidance for international phone numbers #3316
```
Comment (babsdenney) [inline src/_about/whats-new.md:59]: Missing the hyphen

File: src/_about/whats-new.md [hunk 3]
```diff
@@ -25,6 +20,45 @@ title: What’s new?
...
+* [Documentation] Update gender pattern #3289
```
Comment (babsdenney) [inline src/_about/whats-new.md:60]: Missing the hyphen

File: src/_about/whats-new.md [hunk 4]
```diff
@@ -25,6 +20,45 @@ title: What’s new?
...
+* va-select: add import for focusable mixin by @powellkerry in #1376
+* va-modal: set overflow-wrap:anywhere on modal content by @powellkerry in #1372
+* focusable: add tabindex selector by @powellkerry in #1380
+
+### Other changes
+* CSS Library: replace FA icons for action link and alert classes by @jamigibbs in #1371
+* CSS Library: style and image path updates for alert and action link modules by @jamigibbs in #1378
+* CSS Library: remove the /alerts image path by @jamigibbs in #1381
+
+### Design and documentation updates
```
Comment (babsdenney) [inline src/_about/whats-new.md:48]: Maybe we could divide these into documentation updates and design update? @danbrady What do you think? We might have times where something is both a design and documentation update.

File: src/_about/whats-new.md [hunk 5]
```diff
@@ -25,6 +20,45 @@ title: What’s new?
   </div>
 </va-alert>
```
Comment (danbrady) [inline src/_about/whats-new.md:21]: Do we want to remove the alert? If so, we should delete all code up to and including `</va-alert>`.

File: src/_about/whats-new.md (hunk 4 above)
Comment (danbrady) [reply src/_about/whats-new.md:48]: Yeah, I agree. I think we should separate design from documentation.

**Review by danbrady — COMMENTED** [review]
A few minor touch ups needed. We might want to update or remove the global summer updates banner too.

File: src/_about/whats-new.md (hunk 1 above)
Comment (caw310) [reply src/_about/whats-new.md:58]: I will remove [documentation]

File: src/_about/whats-new.md (hunk 3 above)
Comment (caw310) [reply src/_about/whats-new.md:60]: Removed [documentation]

File: src/_about/whats-new.md (hunk 4 above)
Comment (caw310) [reply src/_about/whats-new.md:48]: I've separated design updates and documentation updates.

**Review by caw310 — COMMENTED** [review]
I made updates based on your comments. Thanks!

File: src/_about/whats-new.md (hunk 5 above)
Comment (danbrady) [reply src/_about/whats-new.md:26]: @caw310 Just checking if you saw this comment?

**Review by babsdenney — APPROVED** [review]

**Review by humancompanion-usds — APPROVED** [review]

**Pull request merged**